<launch>

  <!-- Arguments -->
  <arg name="config_path" default="$(find pose_graph)/config/stereo_rig2/stereo_rig_v3_config.yaml" />

  <param name="/use_sim_time" value="true" />


  <!-- To un-compress image topics -->
  <node name="republish1" type="republish" pkg="image_transport" output="screen"
    args="compressed in:=/slave1/image_raw raw out:=/cam0/image_raw" />
  <node name="republish2" type="republish" pkg="image_transport" output="screen"
    args="compressed in:=/slave2/image_raw raw out:=/cam1/image_raw" />


  <!--Run pose_graph node-->
  <!-- <node name="pose_graph_node" pkg="pose_graph" type="pose_graph_node">
    <param name="config_file" type="string" value="$(arg config_path)" />
  </node> -->



  <node name="okvis_node" pkg="okvis_ros" type="okvis_node">

    <param name="config_filename" value="$(find okvis_ros)/okvis/config/config_stereorig_v3.yaml" />
    <param name="mesh_file" value="firefly.dae" />


    <remap from="/camera0" to="/cam0/image_raw" />
    <remap from="/camera1" to="/cam1/image_raw" />

    <remap from="/imu" to="/imu/imu" />

  </node>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find okvis_ros)/config/rviz_svin.rviz" />


</launch>